src/Components/ClinicCardContainer/ClinicCardContainer.js

```diff
@@ -1,19 +1,43 @@
-import React, { useState, useEffect } from "react";
-import ClinicCard from "../ClinicCard/ClinicCard";
-import "./ClinicCardContainer.scss";
+import React, { useState, useEffect } from 'react';
+import ClinicCard from '../ClinicCard/ClinicCard';
+import NoResults from '../NoResults/NoResults';
+import './ClinicCardContainer.scss';
 
-const ClinicCardContainer = ({ states, toggleSelected, filteredResults }) => {
-  const [searchResults, setSearchResults] = useState([]);
+const ClinicCardContainer = ({
+  states,
+  toggleSelected,
+  filteredResults,
+  searchInput,
+}) => {
+  const [authorized, setAuthorized] = useState([]);
+  const [misinformation, setMisinformation] = useState([]);
+  const [resources, setResources] = useState([]);
 
   useEffect(() => {
     if (filteredResults.length > 0) {
```

Comment:
could all of the `state.authorizedClinics.length > 0` in this hook become `state.authorizedClinics.length` and still work?